crates/router/src/core/payouts/validator.rs

```diff
@@ -182,7 +193,100 @@ pub async fn validate_create_request(
         })
         .attach_printable("Profile id is a mandatory parameter")?;
 
-    Ok((payout_id, payout_method_data, profile_id, customer))
+    // fetching payment_method using req.payment_method_id and passing it from here
+    let payment_method: Option<PaymentMethod> =
+        if let Some(payment_method_id) = req.payment_method_id.clone() {
```

Comment:
Let's throw an error if both `payout_method_id` and `payout_token` are passed

Reply:
done